package.json

```diff
@@ -135,7 +135,7 @@
     "fun-hooks": "^0.9.9",
     "gulp-wrap": "^0.15.0",
     "just-clone": "^1.0.2",
-    "live-connect-js": "^6.3.4"
+    "live-connect-js": "^6.6.0"
```

Comment:
Please provide full release notes in your pr description

Reply:
It's added. Please take a look. Thank you!